docs/background.md

```diff
@@ -50,8 +50,7 @@ following statements:
 
   ...
 
-  /* Add Background initializer for HealthKit  */
-  [[RCTAppleHealthKit new] initializeBackgroundObservers:bridge];
+  // [[RCTAppleHealthKit new] initializeBackgroundObservers:bridge]; // <-- Uncomment this line to use background observers
```

Comment:
I don't agree with the changes on this snippet. 

Both commented lines are commented only in the example app, which probably isn't much used as a documentation source. In my opinion, here in the `Background Observer` docs we should just say `add this line to your app's AppDelegate.m file`

Reply:
It makes sense. I just sent another update.